style/js/shield_defs.js

```diff
@@ -798,6 +798,33 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["PH:N"] =
+    shields["PH:N:primary"] =
+    shields["PH:N:secondary"] =
+      {
```

Comment:
```suggestion
  shields["PH:N"] =
      {
```

Let's ship `PH:N` and `PH:E` as initially-supported tagging that has clear community support.